src/controller/proxy/googleapis.controller.ts

```diff
@@ -1,14 +1,18 @@
 import { CacheInterceptor } from '@nestjs/cache-manager';
-import { Controller, Get, UseInterceptors } from '@nestjs/common';
+import { Controller, Get, Query, UseInterceptors } from '@nestjs/common';
 import { GoogleApisService } from 'src/service/googleapis/googleapis.service';
 
+class GetAllocatorsOverviewRequest {
+  tab?: string;
+}
+
```

Comment:
It may make sense to move that to separate file? To the new "dto" directory maybe?